.github/ISSUE_TEMPLATE/e2e_test_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: e2e test report
+about: Report an e2e test failure.
+title: ''
+labels: ['category: e2e', 'needs triage']
+assignees: ''
+
+---
+
+### Description
+<!-- Before reporting an issue, are you sure this e2e test is failing consistently? -->
+<!-- Add a clear and concise description of what the issue is. Please, be as descriptive as possible. -->
+
+**Error**:
+<!-- The error that the e2e test indicates. -->
+
+**Spec**:
+<!-- The test file that's triggering this error. -->
+
+**Relevant line(s) of code**:
+<!-- Line of code that is triggering this error. -->
+
+**GitHub jobs where this spec is failing**:
+<!-- Can you list all the jobs where the test failing? e.g. "WC - latest | wcpay - shopper", "WC - beta | wcpay - shopper" etc. -->
+- 
+
+### Additional context
+<!-- Any additional context or details you think might be helpful. -->
+<!-- Ticket numbers/links, plugin versions, system statuses etc. -->
+
+---
+
+- [ ] Added priority label.
+<!-- For the priority label, use your best judgement. -->
```

Comment:
This a good prompt, I think in many cases the reporter will have a reasonable idea of priority. Possibly even from the test name (i.e. if it's a critical flow or a secondary flow, etc). 

Even better would be to add this as a prompt so reporter can explain their reasoning. For example:

```
### Priority
<!-- add a priority label based on your best judgement (quick is fine) -->
<!-- optional: add comments here to explain your decision or highlight any questions/potential risks -->
````

Reply:
Applied this in https://github.com/Automattic/woocommerce-payments/pull/8420/commits/aa8441ac331c55109d9354eaa70708f05b510598.